backends/tc/tc.def

```diff
@@ -79,61 +80,60 @@ class TCActionParam {
         std::string paramType = "";
         switch(dataType) {
             case TC::BIT_TYPE :
-                paramType += "__u" + Util::toString(bitSize);
+                absl::StrAppend(&paramType, "__u", bitSize);
                 break;
             case TC::DEV_TYPE :
-                paramType += "dev";
+                absl::StrAppend(&paramType, "dev");
                 break;
             case TC::MACADDR_TYPE :
-                paramType += "macaddr";
+                absl::StrAppend(&paramType, "macaddr");
                 break;
             case TC::IPV4_TYPE :
-                paramType += "ipv4";
+                absl::StrAppend(&paramType, "ipv4");
                 break;
             case TC::IPV6_TYPE :
-                paramType += "ipv6";
+                absl::StrAppend(&paramType, "ipv6");
                 break;
             case TC::BE16_TYPE :
-                paramType += "__be16";
+                absl::StrAppend(&paramType, "__be16");
                 break;
             case TC::BE32_TYPE :
-                paramType += "__be32";
+                absl::StrAppend(&paramType, "__be32");
                 break;
             case TC::BE64_TYPE :
-                paramType += "__be64";
+                absl::StrAppend(&paramType, "__be64");
                 break;
         } 
         return paramType;
     }
     toString {
         std::string tcActionParam = "";
-        tcActionParam += "\n\tparam ";
-        tcActionParam += paramName;
-        tcActionParam += " type ";
+        absl::StrAppend(&tcActionParam, "\n\tparam ", paramName.c_str());
```

Comment:
Use `string_view()` method each time you'd need to pass `cstring`'s underlying buffer down to `string_view` argument